public/components/ctas/8.html

```diff
@@ -0,0 +1,9 @@
+<div class="p-20 bg-sky-50 flex items-center flex-wrap justify-between gap-y-8">
+		<div>
+			<p class="text-5xl font-semibold">Ready to grow?</p>
+			<p class="text-5xl font-semibold text-blue-700">Start your free trial now.</p>
+		</div>
```

Comment:
Duplicate `text-5xl font-semibold` should be something like this:

```html
<p class="text-5xl font-semibold">
  Ready to grow?
  
  <em class="text-blue-700 not-italic">Start your free trial now.</em>
</p>
```

Worth checking if `text-5xl` is suitable for mobile devices, I have a feeling it will be too big.